inc/components/namespace.php

```diff
@@ -22,6 +28,32 @@ function get_registry(): Registry {
 	return $registry;
 }
 
+/**
+ * Returns the context object.
+ *
+ * Sets the default 'irving/post' context when first called.
+ *
+ * @return Context_Store The context store object.
+ */
+function get_context_store() {
+	global $wp_query;
+	global $wp_irving_context;
```

Comment:
This needed to be turned into a global so that the defaults can be manipulated during Unit Testing. Unsure if there are other use cases where we would need to reset the defaults during a single application run, but that's why I switched it from a static value.